tests/integration/test_dss.py

```diff
@@ -45,23 +46,63 @@ def test_initialize_creates_dss(cleanup_after_initialize) -> None:
 
     # Check if the dss notebook pvc exists using kubectl
     kubectl_result = subprocess.run(
-        ["kubectl", "get", "pvc", "notebooks", "-n", "dss"], capture_output=True, text=True
+        ["kubectl", "get", "pvc", "notebooks", "-n", DSS_NAMESPACE], capture_output=True, text=True
     )
     assert "notebooks" in kubectl_result.stdout
 
 
-@pytest.fixture()
-def cleanup_after_initialize():
-    """Cleans up resources that might have been deployed by dss initialize.
+def test_create_notebook(cleanup_after_tests) -> None:
+    """
+    Tests that `dss create-notebook` successfully creates a notebook as expected.
+
+    Must be run after `dss initialize`
+    """
+    # TODO: is there a better way to initialize this?  Maybe an optional argument to the test?
+    kubeconfig_file = "~/.kube/config"
+    kubeconfig = lightkube.KubeConfig.from_file(kubeconfig_file)
+    lightkube_client = lightkube.Client(kubeconfig)
+
+    notebook_name = "test-notebook"
+    notebook_image = "kubeflownotebookswg/jupyter-scipy:v1.8.0"
+
+    result = subprocess.run(
```

Comment:
Please add a `FIXME` comment that says we need to remove the `--kubeconfig` option after fixing #37

Reply:
Is this request just for here (the tests) or to remove the option from the command entirely?  I've added a `FIXME` in `main.py` for now with a note about how tests should change too.  Did I interpret this right?

Reply:
I meant remove from the command entirely, this just happens to be here (:

I do not see the FIXME, did I miss something?

Reply:
@DnPlas I see the FIXME is in main.py L54

Reply:
Resolving since there is a fixme where Noha pointed out. Feel free to reopen if we need to add it in every place we use it. However, tests will fail when we remove it given that `kubeconfig` option won't be supported.